### PR TITLE
Module xml extra end tag removed

### PR DESCRIPTION
**Pull request by tejash-wagento**
### Description (*)

Module xml extra end tag removed

### Fixed Issues (if relevant)

1. magento/magento2#26338: Code cleanup for module xml extra end tag removed

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by dmytro-ch — APPROVED** [review]

**Comment by dmytro-ch** [comment]
Hi @tejash-wagento, 
thank you for your contribution!
Please sign CLA, otherwise, we won't be able to process your pull request.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6573](https://jira.corp.magento.com/browse/ENGCOM-6573) has been created to process this Pull Request

File: app/code/Magento/AdvancedPricingImportExport/etc/module.xml [hunk 1]
```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_AdvancedPricingImportExport" >
-    </module>
+    <module name="Magento_AdvancedPricingImportExport" />
```
Comment (lbajsarowicz) [inline app/code/Magento/AdvancedPricingImportExport/etc/module.xml]: XML should not have space before `/>` or `>`

File: app/code/Magento/Dhl/etc/module.xml [hunk 2]
```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_Dhl" >
-    </module>
+    <module name="Magento_Dhl" />
```
Comment (lbajsarowicz) [inline app/code/Magento/Dhl/etc/module.xml]: `Dhl` module depends on `Config` and `Shipping` (quick look).

File: app/code/Magento/Rss/etc/module.xml [hunk 3]
```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_Rss" >
-    </module>
+    <module name="Magento_Rss" />
```
Comment (lbajsarowicz) [inline app/code/Magento/Rss/etc/module.xml]: `Rss` depends on `Config`

File: app/code/Magento/Ups/etc/module.xml [hunk 4]
```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_Ups" >
-    </module>
+    <module name="Magento_Ups" />
```
Comment (lbajsarowicz) [inline app/code/Magento/Ups/etc/module.xml]: `Ups` module depends on `Config` and `Shipping` (quick look).

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Please verify thoroughly the dependencies :-)

File: app/code/Magento/AdvancedPricingImportExport/etc/module.xml (hunk 1 above)
Comment (tejash-wagento) [reply app/code/Magento/AdvancedPricingImportExport/etc/module.xml]: Space removed

File: app/code/Magento/Dhl/etc/module.xml (hunk 2 above)
Comment (tejash-wagento) [reply app/code/Magento/Dhl/etc/module.xml]: Space removed

File: app/code/Magento/Rss/etc/module.xml (hunk 3 above)
Comment (tejash-wagento) [reply app/code/Magento/Rss/etc/module.xml]: Space removed

File: app/code/Magento/Ups/etc/module.xml (hunk 4 above)
Comment (tejash-wagento) [reply app/code/Magento/Ups/etc/module.xml]: Space removed

**Comment by tejash-wagento** [comment]
Hi @lbajsarowicz,

I have resolved issues.

**Comment by lbajsarowicz** [comment]
> Hi @lbajsarowicz,
> 
> I have resolved issues.

Add dependencies I've mentioned :-)

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-6573](https://jira.corp.magento.com/browse/ENGCOM-6573) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**